src/utils.js

```diff
@@ -115,6 +115,18 @@ exports.flattenStartCase = (object) => {
  * > }
  */
 exports.hideAbsolutePathsInObject = (object) => {
+  if (_.isError(object)) {
+
+    // Turn the Error into an Object
+    const error = {};
+
+    _.map(Object.getOwnPropertyNames(object), (key) => {
```

Comment:
Using a map without assigning its result looks weird. Let's either use `_.each`, or convert this into a fold:

```js
const error = _.reduce(Object.getOwnPropertyNames(object), (accumulator, key) => {
    accumulator[key] = object[key]
    return result;
}, {})
```